ArduPlane/GCS_MAVLink_Plane.cpp

```diff
@@ -1137,9 +1137,11 @@ void GCS_MAVLINK_Plane::handle_set_position_target_local_ned(const mavlink_messa
         }
 
         // just do altitude for now
-        plane.next_WP_loc.alt += -packet.z*100.0;
+        plane.next_WP_loc.offset_up_m(-packet.z);
+        float next_WP_loc_relalt = NaNf;
```

Comment:
maybe make it clear with _m ... next_WP_loc_relalt_m

Reply:
Done